BraveRewardsUI/Settings/Tips/Details/TipsDetailViewController.swift

```diff
@@ -94,15 +124,18 @@ extension TipsDetailViewController: UITableViewDataSource, UITableViewDelegate {
     guard let typedSection = Section(rawValue: section) else { return 0 }
     switch typedSection {
     case .summary:
-      return 1
+      return 1 + (tipsList.filter({$0.category == .recurringTip}).isEmpty ? 0 : 1 )
     case .tips:
-//      return 1
-      return 3
+      return tipsList.isEmpty ? 1 : tipsList.count
     }
   }
   
   func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
     tableView.deselectRow(at: indexPath, animated: true)
+    guard Section(rawValue: indexPath.section) == .tips,
```

Comment:
Crashed here when clicking on the empty state cell, needs a check to see if tipsList is non-empty

Reply:
Oh and the empty cell should have `selectionStyle` set to `.none` :)